modules/sdk/src/test/scala/org/tessellation/sdk/infrastructure/collateral/CollateralSuite.scala

```diff
@@ -34,6 +34,8 @@ object CollateralSuite extends MutableIOSuite with Checkers {
   def mkCollateral(balances: Option[Map[Address, Balance]], collateral: Amount)(implicit sc: SecurityProvider[IO]) = {
     val latestBalances = new LatestBalances[IO] {
       def getLatestBalances = IO.delay(balances)
+
+      def getLatestBalancesStream = ???
```

Comment:
`???`

Reply:
It is not needed for this test.

Reply:
Right, I didn't notice that it was a test!